client-datastore-rdbms/src/main/resources/schema-postgresql.sql

```diff
@@ -14,7 +14,7 @@ create table if not exists client (
 
 create index if not exists idx_client_customerId on client (customerId);
 create index if not exists idx_client_customerId_macAddressString on client (customerId, macAddressString);
-
+alter table client add column if not exists macAddressString varchar(100);
```

Comment:
this should go before line 15. 
the point is we want to make sure the macAddressString column exists in the client table before we create an index using that column